### ISSUE TITLE
Acceptance test throws error from `liquid-destination.js`

### ISSUE THREAD

**Issue by zhujy8833** [pzuraq/liquid-wormhole#34]
I've built a modal with `liquid-wormhole`, everything works good on the app side but when I ran some acceptance tests related to the modal, I received error thrown from `liquid-destination.js` pointing to [this line](https://github.com/pzuraq/liquid-wormhole/blob/master/addon/components/liquid-destination.js#L116), which is inside `afterTransition` action.
<img width="748" alt="screen shot 2017-06-15 at 11 21 40 pm" src="https://user-images.githubusercontent.com/1320373/27214544-f4a20ce6-5221-11e7-8569-6a6143632530.png">

<img width="629" alt="screen shot 2017-06-15 at 11 21 52 pm" src="https://user-images.githubusercontent.com/1320373/27214560-fe2bfc0e-5221-11e7-88e1-6e10167aa5a3.png">

The modal component is rendered under a parent route, and the acceptance tests are just verifying open modal and close modal are going to the correct route.

If I set `debugger` around the `this.$(view.element).find('.liquid-wormhole-element').css({ visibility: 'visible' });` I can see sometimes `this.$(view.element)` is undefined. I haven't figured how that happened, but seems simply adding a conditional check `if (this.$(view.element)) {}` is able to fix the issue.

Please help me look into this. Thanks :-)

**Comment by pzuraq** [comment]
hmm, interesting. What version of `liquid-fire` are you using? I'll try to reproduce locally

**Comment by zhujy8833** [comment]
@pzuraq Thanks for following up.
My liquid-fire version is `0.27.3`.
One update is that, it failed only when I use `await` in the test, along with use [wait helper](https://github.com/emberjs/ember-test-helpers/blob/77f9a53da9d8c19a85b3122788caadbcc59274c2/lib/ember-test-helpers/wait.js), 
e.g.
```javascript
await click('#my-button');

wait().then(() => {
  //assertions
});
```
However, if I don't use `await` but the "regular" way, e.g.
```javascript
click('#my-button');

andThen(() => {
  //assertions
});
```
Everything works fine

**Comment by alexspeller** [comment]
fixed by #38
